packages/playground/website/src/lib/resolve-blueprint.ts

```diff
@@ -51,8 +51,10 @@ export async function resolveBlueprint() {
 	// @ts-ignore
 	if (typeof blueprint === 'undefined') {
 		const features: Blueprint['features'] = {};
-		// Networking is enabled by default, so we only need to disable it
-		// if the query param is explicitly set to "no".
+		/**
+		 * Networking is disabled by default, so we only need to enable it
+		 * if the query param is explicitly set to "yes".
+		 */
```

Comment:
Unrelated, but I noticed that the comment was wrong. Networking is disabled by default.